wcivf/apps/people/views.py

```diff
@@ -35,6 +36,10 @@ def get_person(self, queryset=None):
                 'post',
                 'election'
             )
+        try:
```

Comment:
If this is a OneToOneField then I think this is done for us. It should be added to the `select_related` tuple above though, as that will reduce the number of queries on the person pages.